.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+## Issue that this pull request solves
+
+Closes: # (issue number)
+
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+- Files changed
+- Dependencies if any (on other issues/PRs)
+- Basic tests done to validate
+- Conflicts if any (describe the reason for conflict)
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe):
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/mozilla/fxa/blob/master/CONTRIBUTING.md) guidelines
+- [ ] Lint and unit tests pass locally with my changes
```

Comment:
We use lint-staged to lint files before they can be commited, and I believe we also fail CI if linting catches anything, so this might be redundant

Reply:
Cool, I will remove it.

Reply:
I agree we should take out the linting bit, but we could instead rephrase this to be something like, "If applicable, I have modified or added tests which pass locally."